wp-content/themes/pub/wporg-learn-2024/src/style/_sensei.scss

```diff
@@ -73,6 +73,97 @@ body.sensei {
 			font-size: var(--wp--preset--font-size--heading-2);
 		}
 	}
+
+	section.wp-block-sensei-lms-course-outline {
+		> section.wp-block-sensei-lms-course-outline-module-bordered {
+			border: 1px solid var(--wp--preset--color--light-grey-1) !important;
+			border-radius: 2px;
+
+			header {
+				background-color: var(--wp--preset--color--light-grey-2) !important;
+				color: var(--wp--preset--color--charcoal-1) !important;
+				padding: var(--wp--preset--spacing--20);
+
+				h2 {
+					font-size: var(--wp--preset--font-size--heading-6);
+					font-weight: var(--wp--custom--button--typography--font-weight);
```

Comment:
Try to use defined styles as much as possible, and convert those that don't fit semantically to pixels.

Reply:
Is this a button? In most themes we don't use a var for font weight

Reply:
... c701222